src/main/java/io/quarkus/bot/RemoveInvalidLabelOnReopenAction.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.event.Issue;
+import io.quarkiverse.githubapp.event.PullRequest;
+import io.quarkus.bot.config.QuarkusBotConfig;
+import io.quarkus.bot.util.GHIssues;
+import io.quarkus.bot.util.GHPullRequests;
+import io.quarkus.bot.util.Labels;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHPullRequest;
+
+import javax.inject.Inject;
+import java.io.IOException;
+
+class RemoveInvalidLabelOnReopenAction {
+    private static final Logger LOG = Logger.getLogger(RemoveInvalidLabelOnReopenAction.class);
+
+    @Inject
+    QuarkusBotConfig quarkusBotConfig;
+
+    public void onIssueReopen(@Issue.Reopened GHEventPayload.Issue issuePayload) throws IOException {
+        GHIssue issue = issuePayload.getIssue();
+
+        if (GHIssues.hasLabel(issue, Labels.TRIAGE_INVALID)) {
+            System.out.println("has label true");
```

Comment:
Maybe remove this :). Otherwise, it looks good, thanks.

Reply:
ohhh sorry for this one. I removed it.